seqr/views/apis/users_api.py

```diff
@@ -1,5 +1,10 @@
+from __future__ import unicode_literals
+from future.standard_library import install_aliases
+install_aliases()
+
+import urllib.parse as urlparse
```

Comment:
I'm assuming this is a python 3 library support thing, but for my curiosity can you explain why this needs to update?

Reply:
Python 3 has `urllib.parse` while Python 2 doesn't. The code:
```
from future.standard_library import install_aliases
install_aliases()
```
makes Python 2 support `urllib.parse` as well.